dataset_zoo/icdar2013/textrecog.py

```diff
@@ -0,0 +1,67 @@
+data_root = 'data/icdar2013'
+cache_path = 'data/cache'
+
+data_obtainer = dict(
+    type='NaiveDataObtainer',
+    cache_path=cache_path,
+    data_root=data_root,
+    files=[
+        dict(
+            url='https://rrc.cvc.uab.es/downloads/'
+            'Challenge2_Training_Task3_Images_GT.zip',
+            save_name='ic13_textrecog_train_img_gt.zip',
+            md5='6f0dbc823645968030878df7543f40a4',
+            split=['train'],
+            content=['image'],
+            mapping=[['ic13_textrecog_train_img_gt', 'textrecog_imgs/train']]),
+        dict(
+            url='https://rrc.cvc.uab.es/downloads/'
+            'Challenge2_Test_Task3_Images.zip',
+            save_name='ic13_textrecog_test_img.zip',
+            md5='3206778eebb3a5c5cc15c249010bf77f',
+            split=['test'],
+            content=['image'],
+            mapping=[['ic13_textrecog_test_img', 'textrecog_imgs/test']]),
+        dict(
+            url='https://download.openmmlab.com/mmocr/data/mixture/icdar_2013/'
+            'train_label.txt',
```

Comment:
https://download.openmmlab.com/mmocr/data/1.x/recog/icdar_2013/train_labels.json

Reply:
Since the image path is unified to `textrecog_imgs/${split}`. It might be better if we provide this pre-generated JSON file with the same path. So the IC13 data preparer only needs to download data without conversion.

`train/***.png` -> `textrecog_imgs/train/***.png`

Reply:
fixed